e2e/quiz-challenge.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Quiz challenge', () => {
+  test.skip(
+    () => process.env.SHOW_UPCOMING_CHANGES !== 'true',
+    'The FED superblock is not available if SHOW_UPCOMING_CHANGES is false'
+  );
```

Comment:
Just an aside, but this is effectively "skip(() => true)", since we never test upcoming changes in e2e tests.

At some point soonish it would be good if we could run tests against both, but I'm not sure how best to achieve that. We want to avoid just running all the tests again, but with `SHOW_UPCOMING_CHANGES=true`, since it just takes too long.

Reply:
I see what you're saying. We do need a way to test this stuff, otherwise it'll just be a source of false confidence. However, precisely how we should do that isn't obvious to me. Perhaps we need a separate set of tests that run against the upcoming build. It would probably mean wasting a bit more time in CI building both versions of the client, but once the new api is in we can get rid of a few workflows. We should be able to handle the extra build time.